src/schema/users.ts

```diff
@@ -2364,6 +2377,43 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         .update({ id: ctx.userId }, { cover: imageUrl });
       return getCurrentUser(ctx, info);
     },
+    uploadResume: async (
```

Comment:
Where do we check user already uploaded a file, is he supposed to upload multiple times?